cmd/client/client.go

```diff
@@ -42,7 +42,9 @@ var ClientCmd = &cobra.Command{
 		if analyzerAddr != "" {
 			config.Set("analyzers", analyzerAddr)
 		} else {
-			config.SetDefault("analyzers", "localhost:8082")
+			if len(config.GetStringSlice("analyzers")) == 0 {
```

Comment:
why/how SetDefault override the current value of analyzers
Shall SetDefault("analyzers") rather be in config/config.go ?

Reply:
It's probably a bug in Cobra. When doing `config.SetDefault("analyzers", []string{"localhost:8082"})`, it seems to work correctly. I updated the PR